assets/airports/essa.json

```diff
@@ -0,0 +1,2865 @@
+                ["N59d16m36.82", "E017d37m50.70"],
+                ["N59d16m41.64", "E017d37m56.45"],
+                ["N59d16m46.29", "E017d38m02.69"],
+                ["N59d16m50.76", "E017d38m09.39"],
+                ["N59d16m55.05", "E017d38m16.54"],
+                ["N59d16m59.15", "E017d38m24.13"],
+                ["N59d17m03.03", "E017d38m32.12"],
+                ["N59d17m06.70", "E017d38m40.49"],
+                ["N59d17m10.13", "E017d38m49.24"],
+                ["N59d17m13.33", "E017d38m58.32"],
+                ["N59d17m16.28", "E017d39m07.71"],
+                ["N59d17m18.97", "E017d39m17.40"],
+                ["N59d17m21.40", "E017d39m27.35"],
+                ["N59d17m23.56", "E017d39m37.53"],
+                ["N59d17m25.45", "E017d39m47.92"],
+                ["N59d17m27.05", "E017d39m58.49"],
+                ["N59d17m28.37", "E017d40m09.21"],
+                ["N59d17m29.40", "E017d40m20.05"],
+                ["N59d17m30.13", "E017d40m30.98"],
+                ["N59d17m30.58", "E017d40m41.97"],
+                ["N59d17m30.72", "E017d40m53.00"]
+            ]
+        }
+    ],
+    "_comment": "Source runways: https://www.aro.lfv.se/Editorial/View/7952/ES_AD_2_ESSA_en",
+    "runways": [
+        {
+            "name": ["01L", "19R"],
+            "end":    [["N59d38m14.11", "E017d54m47.60", "98.6ft"], ["N59d39m59.04", "E017d55m25.56", "118.2ft"]],
+            "ils": [true, true]
+        },
+        {
+            "name": ["01R","19L"],
+            "end": [["N59d37m35.03", "E017d57m02.67", "138.0ft"], ["N59d38m54.49", "E017d57m31.48", "98.9ft"]],
+            "ils": [true, true]
+        },
+        {
+            "name": ["08", "26"],
+            "end": [["N59d39m30.31", "E017d56m10.08", "108.0ft"], ["N59d39m50.03", "E017d58m44.96", "124.8ft"]],
+            "ils": [false, true]
+        }
+    ],
+    "_comment": "Source runway 01L SIDs: https://www.aro.lfv.se/Editorial/View/7944/ES_AD_2_ESSA_4-9_en",
+    "_comment": "Source runway 01R SIDs: https://www.aro.lfv.se/Editorial/View/7945/ES_AD_2_ESSA_4-15_en",
+    "_comment": "Source runway 08 left turn SIDs: https://www.aro.lfv.se/Editorial/View/7946/ES_AD_2_ESSA_4-19_en",
+    "_comment": "Source runway 08 right turn SIDs: https://www.aro.lfv.se/Editorial/View/7947/ES_AD_2_ESSA_4-25_en",
+    "_comment": "Source runway 19L SIDs: https://www.aro.lfv.se/Editorial/View/7948/ES_AD_2_ESSA_4-31_en",
+    "_comment": "Source runway 19R SIDs: https://www.aro.lfv.se/Editorial/View/7577/ES_AD_2_ESSA_4-37_en",
+    "_comment": "Source runway 26 SIDs: https://www.aro.lfv.se/Editorial/View/7578/ES_AD_2_ESSA_4-41_en",
+    "sids": {
+        "AROS5C": {
+            "icao": "AROS5C",
+            "name": "Aros Five Charlie",
+            "altitude": 5000,
+            "rwy": {
+                "ESSA01L": []
+            },
+            "body": [["^SA421", "A06+"], ["SA850", "A15+|S205-"], "SA851", "SA723", "ARS"],
+            "exitPoints": {
+                "ARS": ["ARS"]
```

Comment:
I guess ARS doesn't need to be (probably even wrong) both in the body and as an exitPoint? I just did it this way for now and I'll change all of the SIDs at the same time based on the response to this comment.

Reply:
I removed duplicates from the end of the body so they are only  in the exitPoints. Will push later!